server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/review/PullRequestReview.java

```diff
@@ -0,0 +1,59 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest.review;
+
+import java.time.OffsetDateTime;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.lang.NonNull;
+
+import de.tum.in.www1.hephaestus.codereview.base.BaseGitServiceEntity;
+import de.tum.in.www1.hephaestus.codereview.comment.review.ReviewComment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.PullRequest;
+import de.tum.in.www1.hephaestus.codereview.user.User;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "pull_request_review")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(callSuper = true)
+public class PullRequestReview extends BaseGitServiceEntity {
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "author_id")
+    @ToString.Exclude
+    private User author;
+
+    @NonNull
+    private PullRequestReviewState state;
+
+    private OffsetDateTime createdAt;
```

Comment:
Can't you remove `createdAt` and `updatedAt` in this entity since `BaseGitServiceEntity` already contains it? Or do you always have to include it in the child entities?

Reply:
You are absolutely correct, the parent already contains them. Of course we shouldn't create them twice.